actions/publish-image/action.yaml

```diff
@@ -106,10 +106,20 @@ inputs:
     default: true
     type: boolean
 
+  push-to-prime-staging:
+    description: |
+      Indicates whether the image should be pushed to the Prime Staging registry.
+    default: false
+    type: boolean
+
```

Comment:
I think the description could use a bit more detail about the behavior, particularly around tag parsing for release candidates.